rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -152,6 +155,93 @@ void voteFeePerKbChange_success() {
         assertEquals(FeePerKbResponseCode.SUCCESSFUL_VOTE.getCode(), result);
     }
 
+    @Test
+    void getLockWhitelistSize() {
+        WhitelistSupport whitelistSupport = mock(WhitelistSupportImpl.class);
+        when(whitelistSupport.getLockWhitelistSize()).thenReturn(Integer.SIZE);
+
+        BridgeSupport bridgeSupport = bridgeSupportBuilder
+            .withWhitelistSupport(whitelistSupport)
+            .build();
+
+        assertEquals(Integer.SIZE, bridgeSupport.getLockWhitelistSize());
+    }
+
+    @Test
+    void getLockWhitelistEntryByIndex() {
+        WhitelistSupport whitelistSupport = mock(WhitelistSupportImpl.class);
+        LockWhitelistEntry entry = mock(LockWhitelistEntry.class);
+        when(whitelistSupport.getLockWhitelistEntryByIndex(Integer.SIZE)).thenReturn(entry);
```

Comment:
If the max size is `Integer.SIZE` wouldn't this get by index cause an out of bounds exception? Not sure if I am understanding this correctly.

Reply:
I used `Integer.SIZE` to avoid using, for instance, a raw number like 0(zero). About getting an exception, in this case, we are not going to get any exception since we're mocking. Please let me know if `Integer.SIZE` might confuse the sense of the test, I could change it for something like this:

`when(whitelistSupport.getLockWhitelistEntryByIndex(0)).thenReturn(entry);`

Reply:
Thank you for your feedback. Yes, I believe using a value such as 0 directly is less confusing in this context.

Reply:
Done!